app/controllers/registration_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RegistrationController < ApplicationController
```

Comment:
In the future, I would recommend using `git mv` to rename files; otherwise, git thinks this is a new file when in reality it's just been renamed. That way, we can see the differences between this one and the one above it. Is there anyway you can revert the name change and then use `git mv`?